recipes/datreant/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: datreant
+  version: 1.0
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
Yes we using file-locking modules which aren't cross platform. see [issue #93](https://github.com/datreant/datreant.core/issues/93)